src/components/EntityInfo/EntityMenu.vue

```diff
@@ -99,7 +99,7 @@
         <v-list dense>
           <v-list-item-group color="primary">
             <!-- View/Add Digital Credentials -->
-            <span v-if="isAllowed(AllowableActions.DIGITAL_CREDENTIALS)">
+            <span v-if="isAllowed(AllowableActions.DIGITAL_CREDENTIALS) && !isDCRoute">
```

Comment:
I see you're hiding just the DC menu item.

I think you should hide the entire menu, to make the user feels like they "in" a different section of the app, otherwise it seems odd to show the other menu items.

The user can always return to the business dashboard (main view) using the breadcrumbs, or perhaps you should provide a "return" or "cancel" button on your cred dashboard?

I can put you in touch with one of our UI designers if you want another opinion.

Reply:
I can do that if it makes sense to

Reply:
Yes, please. I just double-checked with our UI designer (Yui) and he says the menu bar should be hidden when on the DC pages. He also says users will probably click the breadcrumb (<-) button to return to the main dashboard and that's fine; no need for another return or cancel button.

Reply:
Fixed, thanks.